tests/includes/utils.php

```diff
@@ -66,14 +66,17 @@ static function change_it( $var ) {
         return rand_str();
     }
 }
- 
+
 /**
 * var_dump() for Unit Tests
 *
 * @since 0.1
 */
-function ut_var_dump( $what ) {
+function ut_var_dump( ...$what ) {
```

Comment:
Wow, is it compatible with 5.6?

Reply:
Yes, it is: https://www.php.net/manual/en/functions.arguments.php#functions.variable-arg-list